rpm/vim.spec

```diff
@@ -113,13 +121,14 @@ export CXXFLAGS="$RPM_OPT_FLAGS -D_GNU_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_
 
 sed -i 's/-l\$with_tlib/-ltinfo -l\$with_tlib/g' configure
 sed -i 's/-l\$with_tlib/-ltinfo -l\$with_tlib/g' auto/configure
+export SOURCE_DATE_EPOCH="unknown"
 
 %configure --prefix=%{_prefix} --with-features=huge --enable-pythoninterp \
  --enable-perlinterp --disable-tclinterp --with-x=no \
  --enable-gui=no --exec-prefix=%{_prefix} --enable-multibyte \
- --enable-cscope --with-modified-by="<bugzilla@meego.com>" \
+ --enable-cscope \
  --with-tlib=ncurses \
- --with-compiledby="<bugzilla@meego.com>" \
+ --with-compiledby="OBS" \
```

Comment:
Is this really needed here? I would cut out if not, otherwise I would make this %vendor or %_vendor that returns the platform if possible

Reply:
If not included my local build added "Compiled by pvuorela", assuming obs gets some similar value included too. 

%_vendor seems to be there, but resolving now to "meego" which isn't much of an improvement to the end result. If you feel that is better I can change it in the hope of some day having a better value there, but all in all this is a drive-by adjustment.

Reply:
Almost all of our packages end up having vendor=meego. So I'd consider having it happen to this package too to be a good thing in the sense that it is consistent and there presumably is some single control point that could be used to change vendor in all packages.

... I have vendor override for local package builds -> makes it easy to check if I have self-built custom packages installed via e.g. rpm -qa --queryformat '%{VENDOR} %{NAME}\n'| sort -u| grep -v ^meego

Reply:
If not passed this will be empty:
https://github.com/vim/vim/blob/6079da7cfb10c798535224c92db7fedd3914cb71/src/version.c#L9328

Reply:
Looks like getting set in src/Makefile (yea, really), and not much chance to escape it:
```
        -@echo 'char_u *compiled_user = (char_u *)"' | tr -d $(NL) >> $@
        -@if test -n "$(COMPILEDBY)"; then \
                echo "$(COMPILEDBY)" | tr -d $(NL) >> $@; \
                else ((logname) 2>/dev/null || whoami) | tr -d $(NL) >> $@; fi
        -@echo '";' >> $@
        -@echo 'char_u *compiled_sys = (char_u *)"' | tr -d $(NL) >> $@
        -@if test -z "$(COMPILEDBY)"; then hostname | tr -d $(NL) >> $@; fi
        -@echo '";' >> $@
```

This is braindead.

For "meego", at least it's better than current bugzilla email, but as visible in the app I'd consider it worse than the common rpm vendor info.

Reply:
Ok, %_vendor it is for now. At least it's not hard-coded to broken email address.